File: 2.JS/3.js
/*26. Calcular se um número é ou não primo*/

let num = 29;
let contagem = 0;

for (let i = 1; i <= num; i++) {
    if (num % i === 0) {
        contagem++;
    }
}

if (contagem === 2) {
    console.log(num + " é primo");
} else {
    console.log(num + " não é primo");
}

File: 1.C/3.c
/*17. Calcular se um número é ou não primo*/

#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, "Portuguese");

    int NUM, i, CONTAGEM = 0;

    printf("Número: ");
    scanf("%d", &NUM);

    for (i = 1; i <= NUM; i++)
    {
        if (NUM % i == 0)
        {
            CONTAGEM++;
        }
    }
    if (CONTAGEM == 2)
    {
        printf("%d é primo\n", NUM);
    }
    else
    {
        printf("%d não é primo\n", NUM);
    }
}